tests/tx/test_tips.py

```diff
@@ -0,0 +1,182 @@
+from hathor.transaction import Transaction
+from tests import unittest
+from tests.utils import (
+    add_blocks_unlock_reward,
+    add_new_block,
+    add_new_blocks,
+    add_new_double_spending,
+    add_new_transactions,
+)
+
+
+class BaseTipsTestCase(unittest.TestCase):
+    __test__ = False
+
+    def setUp(self):
+        super().setUp()
+
+        self.network = 'testnet'
+        self.manager = self.create_peer(self.network, unlock_wallet=True)
+
+    def test_tips_back(self):
+        add_new_block(self.manager, advance_clock=1)
+        add_blocks_unlock_reward(self.manager)
+        self.assertEqual(len(self.manager.tx_storage._tx_tips_index), 0)
```

Comment:
I feel all these `_tx_tips_index` should be renamed to `_tx_mempool_index `, right?

Reply:
Renamed.

Reply:
Great! Thanks!